src/styles/partials/components/_collapse.scss

```diff
@@ -33,5 +34,6 @@
 
   .dg_accordion_buttontext-holder {
     margin-left: $margin-large;
+    margin-top: $margin-large;
```

Comment:
This rule isn't doing anything because this element displays is inline.

I think that's ok thought because this looks right to me as is. In this case I think the padding is doing ok in this instance?


![image](https://user-images.githubusercontent.com/1933400/73290848-18e54080-41cd-11ea-8d15-f83d8638e8da.png)


Maybe there is something I'm missing?

Reply:
No that was a mistake and has been removed